code/modules/recycling/disposal-unit.dm

```diff
@@ -40,6 +43,8 @@
 	//gas.volume = 1.05 * CELLSTANDARD
 	update_icon()
 
+	return INITIALIZE_HINT_LATELOAD //we need turfs to have air
```

Comment:
The way this works is it asks SSatoms to call atom/LateInitialize() after it has Initialized all atoms. You didn't define one

Reply:
No it exists, I just moved this line from a duplicate Initialize